packages/typespec-test/test/openai_modular/spec/tspconfig.yaml

```diff
@@ -5,6 +5,7 @@ options:
     azureSdkForJs: false
     isModularLibrary: true
     hierarchyClient: false
+    generateOrphanModels: true
```

Comment:
Is there any documentation we need to update for this new setting?

Reply:
added here https://github.com/Azure/autorest.typescript/pull/2356/files#diff-0dc6d1b55dc9a9970d589d1beb757622461950f62cd6107358e29fc205f04f8dR148